openapi/azure.openai.chat/original-openapi.yaml

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
I'm seeing 2 openAPI files being added as `openapi.yaml` and `original-openapi.yaml`. Why do we need to keep both?

Reply:
We are splitting a spec into 3 to generate 3 separate connectors. As discussed in a meeting regarding this, we decided to keep the original spec in each connector directory (similar to how it is in the Stripe connector). This is mainly done to be able to trace the versions of the original spec when they get updated

Reply:
@JDPrabasha I see.. makes sense